parsers/to-react-native/tokens/gradient.ts

```diff
@@ -0,0 +1,26 @@
+import tinycolor from 'tinycolor2';
+import { GradientToken, StepForGradient } from '../../../types';
+import { OptionsType } from '../to-react-native.parser';
+
+export class Gradient extends GradientToken {
+  constructor(token: Partial<GradientToken>) {
+    super(token);
+  }
+
+  toReactNative({ colorFormat = 'rgb' }: OptionsType = {}) {
+    const value: any[] = this.value.gradients.map(gradient => {
```

Comment:
Prefer the usage of typescript inference instead of `any` type.
You can remove `any[]` safely.

Reply:
Whoops that's lazy on my behalf 🙈